Examples.md

```diff
@@ -184,10 +184,7 @@ spec:
                  - name: clickhouse-backup
                    image: altinity/clickhouse-backup:master
```

Comment:
Could you change `:master` to `:latest` in whole Examples.md?

Reply:
Done! 👍